client/Dockerfile

```diff
@@ -19,6 +19,9 @@
 ARG PROJECT_ID=YOURPROJECTID
 FROM gcr.io/$PROJECT_ID/firebase
 
+# Override with "--build-arg AVOCANO_PURCHASE_MODE=cart"
+ARG AVOCANO_PURCHASE_MODE=""
```

Comment:
Silly question, is there a way to initialize env vars from build args in Dockerfile? 
I can also make a script available in package.json that would just run `AVOCANO_PURCHASE_MODE=cart npm run start`.